src/main/include/log4cxx/net/telnetappender.h

```diff
@@ -80,12 +61,9 @@ class LOG4CXX_EXPORT TelnetAppender : public AppenderSkeleton
 		~TelnetAppender();
 
 		/**
-		This appender requires a layout to format the text to the
-		attached client(s). */
-		bool requiresLayout() const override
-		{
-			return true;
-		}
```

Comment:
Note: I'm pretty sure changing the inline function to be not inline would cause an ABI version bump, but since this is only used in log4cxx code I think it's safe to do.